bam2fastx.wdl

```diff
@@ -38,6 +38,21 @@ task Bam2Fasta {
     command {
         set -e
         mkdir -p "$(dirname ~{outputPrefix})"
+
+        # Localise the bam and pbi files so they are next to each other in the
+        # current folder.
+        bamFiles=""
+        for bamFile in ~{sep=" " bam}
+        do
+            ln $bamFile .
+            bamFiles=$bamFiles" $(basename $bamFile)"
+        done
+
+        for index in ~{sep=" " bamIndex}
+        do
+            ln $index .
+        done
+
```

Comment:
Isn't this repeated from below?

Reply:
It is indeed the same as below. This new entry is part of the `Bam2Fasta` task. The below is `Bam2Fastq`.
Previously, only the `Bam2Fastq` task had this localization. But the `Bam2Fasta` requires it as well.

Reply:
Oh it's a different task XD